packages/frontapp/src/app/components/BookCardExtended/index.spec.tsx

```diff
@@ -0,0 +1,72 @@
+import { render } from '../../../helpers/customRender';
+import BookCardExtended from './index';
+import { IMaterial } from '../../types';
+import { StatusTypes } from '@mimir/global-types';
+import { screen } from '@testing-library/react';
+
+describe('render BookCardExtended', () => {
+  it('correctly render with status is free', () => {
+    const mockItem: IMaterial | null = {
```

Comment:
Hmm. I don't get how `const` could possible be `null` when the value set at initialization.